### PR TITLE
Add stack clash protection by default

### PR DESCRIPTION
**Pull request by kgaillot**
... along with a minor RPM improvement and some updated URLs in the documentation

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this when you get a chance? Thanks

**Comment by clumens** [comment]
This all looks good to me.

**Comment by kgaillot** [comment]
@clumens, could you review the two additional commits I couldn't resist adding? :)

File: README.markdown [hunk 1]
```diff
@@ -13,7 +13,7 @@ resource health.
 
 ## Who is Pacemaker?
 
-Pacemaker is distributed by [ClusterLabs](http://www.clusterlabs.org).
+Pacemaker is distributed by [ClusterLabs](https://www.clusterlabs.org/).
```
Comment (clumens) [inline README.markdown:16]: While you're at it, there's a bunch of other references to "http://www.clusterlabs.org" - all the conf.py files in doc/sphinx, several in doc/crm_fencing.html, and one in extra/cluster-init.

Comment (kgaillot) [reply README.markdown:16]: Good catch, replaced last commit

The conf.py are created from conf.py.in, and crm_fencing.html is created from crm_fencing.txt. crm_fencing.html and cluster-init are woefully outdated overall, but that will have to wait for another time ...

**Comment by kgaillot** [comment]
References to book URLs will have to be updated again once the sphinx docs are uploaded, but that will wait too

**Pull request merged**